### DISCUSSION TITLE
making Debian an official platform for RKE2?

### DISCUSSION THREAD

**Discussion by harridu** [rancher/rke2#4888] [General]
RKE2 seems to work very well on Debian 11 and 12, so I wonder why it is not [officially supported](https://docs.rke2.io/install/requirements/#operating-systems)? According to https://www.enterpriseappstoday.com/stats/linux-statistics.html Debian is on position 2 in market share (behind Debian-derived Ubuntu and ahead of Centos/Redhat EL).  Esp Debian is much more popular for running [web services](https://w3techs.com/technologies/comparison/os-debian,os-redhat) than RedHat EL.

I am not asking to drop any of these, of course.

**Comment by wangdong-sd** [comment]
yeah, I got the same question here. Why ubuntu over debian? WHY!
